Specification/8._Hardware_design_recommendations.md

```diff
@@ -4,22 +4,22 @@ permalink: /Specification/8._Hardware_design_recommendations/
 
 # Hardware design recommendations
 
-The points concerning hardware design practices are listed in this chapter.
-Following them is highly recommended, because that would ensure the highest level of inter-vendor compatibility
-and would allow the developers to avoid some of the common pitfalls.
+This chapter contains best-practice recommendations for hardware design.
+
+Following these guidelines will ensure the highest level of inter-vendor compatibility
+and allow the developers to avoid many common design pitfalls.
 
 ## Physical connector definition
 
+**Do not deviate from the recommendations provided in this section, as this may impact cross-vendor compatibility.**
+
 {% include lightbox.html url="/Specification/figures/connector_size_comparison.jpg" title="Size comparison of different UAVCAN connector types. The blue PCB is a small UAV avionic device given for scale." thumbnail=true %}
```

Comment:
Image might be improved with a little whitespace between the different connectors. Looks a bit of a mess as it is currently.

Reply:
@j3qq4hch could you please provide us with a better image, if you still have the model? If you don't, that's alright too.

Reply:
![3d_view](https://user-images.githubusercontent.com/9300016/32652733-5a5801ae-c616-11e7-8732-310d39a1440d.png)
Here it is

Reply:
@j3qq4hch Thanks, Alex. Sorry for not telling you earlier, but could you please get rid of the M5 connector (we don't support this type), and also add comments describing which connector is where? Like this:

![image](https://user-images.githubusercontent.com/3298404/32660929-8999d372-c62d-11e7-8df6-36050c4cea0f.png)

Thanks in advance.

Reply:
![3d_view](https://user-images.githubusercontent.com/9300016/32661540-5eae9fac-c638-11e7-926f-2d68d3b86598.png)

Reply:
Sorry to hear about M5. I liked it

Reply:
Thanks @j3qq4hch That looks great. Doc updated.